iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/common/import/oozie_app/workflow.xml

```diff
@@ -710,16 +718,26 @@
         <java>
 			<prepare>
```

Comment:
Strange indentation. This whole file should be reindented to make the indentation uniform.